### DISCUSSION TITLE
USE TERRIA JS FOR LUNAR (MOON)

### DISCUSSION THREAD

**Discussion by lakhwaniji** [TerriaJS/terriajs#7192] [General]
Well I am wondering that can I use Terria Map and modify it in such way that I can use it for lunar surface and if yes how can I change the earth to moon

**Comment by Techmaven-geospatial** [comment]
sure just create some imagery and quantized mesh for the moon and load some data.
do you have access do any lunar data?

**Reply by lakhwaniji** [reply]
Well I have data but I want to know one thing that I want to show the moon on 3d sphere in place of earth for that do I need an wms server or there can be any other way.

**Reply by lakhwaniji** [reply]
So what I want is that earth Mao is replaced by moon and then the user can upload the data related to moon and can see the points or other data for the moon, How can I do that and for this do I need to change the ClientSideConfig of Terria Map or I have to modify Terria JS
